main.css

```diff
@@ -4,5 +4,4 @@ body {
 
 h1 {
   font-style: italic;
-  color: lightblue;
```

Comment:
I think it would have been nicer with the dark blue, instead of going back to black.